SimMuon/Configuration/python/SimMuon_EventContent_cff.py

```diff
@@ -49,6 +49,7 @@
 phase2_muon.toModify( SimMuonFEVTDEBUG, outputCommands = SimMuonFEVTDEBUG.outputCommands + ['keep *_simMuonME0PseudoDigis_*_*',
                                                                                             'keep *_simMuonME0PseudoReDigis_*_*',
                                                                                             'keep *_simMuonME0Digis_*_*',
-                                                                                            'keep *_simMuonME0PadDigis_*_*'] )
+                                                                                            'keep *_simMuonME0PadDigis_*_*',
+                                                                                            'keep *_simMuonME0PadDigiClusters_*_*'] )
```

Comment:
Do you have an estimate of how much space the additional collection uses? (I imagine it's negligible, but always good to check)